File: Examples/Mvc.VB/Models/OverviewViewModel.vb
Imports GleamTech.ExamplesCore

Namespace Models

    Public Class OverviewViewModel
        Public Sub New()
            ImageData = New Dictionary(Of String, Object)()
            ImageExifMetadata = New Dictionary(Of String, Tuple(Of String, String))()
            ImageIptcMetadata = New Dictionary(Of String, Tuple(Of String, String))()
        End Sub

        Public Property ExampleFileSelector As ExampleFileSelector

        Public Property ImageData As Dictionary(Of String, Object)

        Public Property ImageExifMetadata As Dictionary(Of String, Tuple(Of String, String))

        Public Property ImageIptcMetadata As Dictionary(Of String, Tuple(Of String, String))

        Public Property ImagePath As String
    End Class

End NameSpace

File: Examples/AspNetMvcCS/Models/OverviewViewModel.cs
using System;
using System.Collections.Generic;
using GleamTech.Examples;

namespace GleamTech.ImageUltimateExamples.AspNetMvcCS.Models
{
    public class OverviewViewModel
    {
        public OverviewViewModel()
        {
            ImageData = new Dictionary<string, object>();
            ImageExifMetadata = new Dictionary<string, Tuple<string, string>>();
            ImageIptcMetadata = new Dictionary<string, Tuple<string, string>>();
        }

        public ExampleFileSelector ExampleFileSelector { get; set; }

        public Dictionary<string, object> ImageData { get; set; }

        public Dictionary<string, Tuple<string, string>> ImageExifMetadata { get; set; }

        public Dictionary<string, Tuple<string, string>> ImageIptcMetadata { get; set; }

        public string ImagePath { get; set; }
    }
}